Dockerfile

```diff
@@ -0,0 +1,48 @@
+# RethinkRobotics/baxter-simulator:kinetic
+# Run simulated Baxter in Gazebo
+
+FROM osrf/ros:kinetic-desktop-full
+MAINTAINER Dave Coleman dave@dav.ee
+
+ENV TERM xterm
+
+# Setup catkin workspace
+ENV CATKIN_WS=/root/ws_baxter
+RUN mkdir -p $CATKIN_WS/src
+WORKDIR $CATKIN_WS/src
+
+# Download source code
+RUN wstool init . && \
+    wstool merge https://raw.githubusercontent.com/RethinkRobotics/baxter_simulator/${ROS_DISTRO}-gazebo7/baxter_simulator.rosinstall && \
```

Comment:
Currently, this is the link:
https://raw.githubusercontent.com/RethinkRobotics/baxter_simulator/kinetic-devel/baxter_simulator.rosinstall